tests/test_app/src/Model/Endpoint/TestEndpoint.php

```diff
@@ -32,7 +32,7 @@ public function validationDefault(Validator $validator): Validator
      *
      * @return true
      */
-    public function findExamples()
+    public function findExamples(): true
```

Comment:
```suggestion
    public function findExamples(): bool
```
`true` return type is available since PHP 8.2 only

Reply:
Ahh, I was about to install php 8.1 to test this locally. Will still install it though ;). Thanks!